src/linear_algebra/std_basis.lean

```diff
@@ -251,10 +251,22 @@ by { simp only [basis_fun, basis.coe_of_equiv_fun, linear_equiv.refl_symm,
   (pi.basis_fun R η).repr x i = x i :=
 by simp [basis_fun]
 
-/-- The standard basis on `matrix n m R`. -/
+/-- The standard basis of `matrix n m R`. -/
 noncomputable def _root_.matrix.std_basis (n : Type*) (m : Type*) [fintype m] [fintype n] :
-  basis (Σ (j : n), (λ (i : n), m) j) R (matrix n m R) :=
-pi.basis (λ (i : n), pi.basis_fun R m)
+  basis (n × m) R (matrix n m R) :=
+basis.reindex (pi.basis (λ (i : n), pi.basis_fun R m)) (equiv.sigma_equiv_prod _ _)
+
+lemma matrix.std_basis_eq_std_basis_matrix {n : Type*} {m : Type*} (i : n) (j : m)
```

Comment:
This will have its name start with  `pi`.

With the suggestion above:
```suggestion
lemma std_basis_eq_std_basis_matrix {n : Type*} {m : Type*} (i : n) (j : m)
```
otherwise:
```suggestion
lemma _root_.matrix.std_basis_eq_std_basis_matrix {n : Type*} {m : Type*} (i : n) (j : m)
```